src/PostgREST/Config.hs

```diff
@@ -42,15 +42,16 @@ import           Data.Scientific              (floatingOrInteger)
 import           Data.String                  (String)
 import           Data.Text                    (dropAround,
                                                intercalate, lines,
-                                               strip)
+                                               strip, take)
 import           Data.Text.Encoding           (encodeUtf8)
 import           Data.Text.IO                 (hPutStrLn)
 import           Data.Version                 (versionBranch)
+import           Development.GitRev
```

Comment:
maybe an explicit import here of `gitHash`

Reply:
Good idea